### DISCUSSION TITLE
Two left drawer side-by-side

### DISCUSSION THREAD

**Discussion by 1Miguel** [zauberzeug/nicegui#4065] [Q&A] [answered]
### Question

How do I place two left drawer side-by-side?

I'm getting one of my side drawer overlapping with my existing side drawer.

![image](https://github.com/user-attachments/assets/d13da4b8-374f-435a-9600-915b8f707e6f)

**Comment by falkoschindler** [comment, marked as answer]
Hi @1Miguel,

Borrowing the CSS from https://stackoverflow.com/a/77115536/3419103, you can do something like this:

```py
ui.add_css('''
    .q-drawer-container:nth-child(1) > .q-drawer--left {
        margin-left: 0px !important;
    }
    .q-drawer-container:nth-child(2) > .q-drawer--left {
        margin-left: 300px !important;
    }
''')

with ui.left_drawer().classes('bg-blue-100'):
    ui.label('Drawer 1')

with ui.left_drawer().classes('bg-green-100'):
    ui.label('Drawer 2')
```
<img width="1228" alt="Screenshot 2024-12-04 at 10 23 40" src="https://github.com/user-attachments/assets/310e594a-e185-45a6-ad11-73a85dafec34">

**Reply by 1Miguel** [reply]
Thanks!!! I already saw that stack overflow but I can't make it work, didn't know that it was that simple!
